browserlist

```diff
@@ -0,0 +1,8 @@
+# See https://support.bigcommerce.com/articles/Public/Themes-Supported-Browsers/
```

Comment:
which is the source of truth here?  Themes-Supported-Browsers or the following in config.json:

```
  "autoprefixer_browsers": [
    "last 3 versions"
  ],
```